swimster-ui/src/components/Login/Login.jsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react'
+import { Link } from 'react-router-dom'
+import { useLoginForm } from '../../hooks/useLoginForm'
+import { style } from './style'
+
+const Login = () => {
+    const { form, error, isProcessing, register, handleSubmit, onSubmit } = useLoginForm()
+
+    return (
+        <div className={style.register}>
+            <div className={style.formContainer}>
+                <h1 className={style.formHeader}>Welcome Back!</h1>
+                <br></br>
+                <form className={style.userDetails} onSubmit={handleSubmit(onSubmit)}>
+                    <input
+                        className={
+                            `${style.inputElement} ${style.fullInput}`
+                        }
+                        type="email"
+                        placeholder="Email"
+                        {...register("email")}
+                    />
+                    <input
+                        className={
+                            `${style.inputElement} ${style.fullInput}`
+                        }
+                        type="password"
+                        placeholder="Password"
+                        {...register("password")}
+                    />
+                    <br></br>
```

Comment:
nit: next time please use a margin instead of hardcoding a white space! it's cleaner that way

Reply:
oh yeah I should do that